mautic/mautic
# Faster leads list page

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Y
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

The leads list page is slow and takes more than 30 seconds to load when the database include lots of IP addresses, this PR removes the IP addresses table from the query that is used to list the leads list ~~since they IP addresses are not used ~~



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Have 10 contacts with a large number of ipaddresses related to them
3. Check the contacts page taking long time to load
4. Apply the PR and check it loads faster
